features/facebookImport/src/model/analysis.js

```diff
@@ -1,8 +1,5 @@
 import { ZipFile } from "../model/storage.js";
-import {
-    createErrorStatus,
-    createSuccessStatus,
-} from "./analyses/utils/analysis-status.js";
+import { createErrorStatus } from "./analyses/utils/analysis-status.js";
```

Comment:
`createSuccessStatus` elimination is simply because it wasn't used anyway? Or is related to the creation of the cache?